library/model/src/user/cms_user.e

```diff
@@ -80,6 +80,11 @@ feature -- Access
 			--			active
 			--			trashed
 
+	application: detachable STRING_32
```

Comment:
No please, do not add this to CMS_USER

if needed, we can use CMS_USER_PROFILE for that purpose.
we should try to keep CMS_USER as light as possible.